### PR TITLE
Vikram readme

### PR DESCRIPTION
**Pull request by VikramBombhi**
Updated the readme nothing special

### CODE REVIEW

**Comment by VikramBombhi** [comment]
and yes I will squash all commits before merging

**Comment by JackyChiu** [comment]
For people reading PR:

CU-News
===================

CU-News is a web app that displays news articles from multiple reputable sources and finds which news articles Carleton students care about, with analysis of trends.

screenshot of CU-News


----------

Voting System
-------------

Each article has key phrases extracted from it and upvoting a article adds points to all key phrases associated to that article. Popular key phrases show us issues for topics people care most about.

----------

Analytics
-------------------

The popularity of all key phrases can be graphed over time and can be compared to each other

> **Note:**

> - Currently only the top key phrase is being graphed at the top of the page
> - Currently only a line graph can be displayed
> - Currently time is shown in unix time

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-# Wall Of Memes
+CU-News
+===================
+
+CU-News is a web app that displays news articles from multiple reputable sources and finds which news articles Carleton students care about, with analysis of trends.
+
+![screenshot of CU-News](http://i.imgur.com/L2xU1LM.png)
+
+
+----------
```
Comment (JackyChiu) [inline README.md]: Most readme's don't have this, we can lose it unless you like it

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+Voting System
+-------------
+
+Each article has key phrases extracted from it and upvoting a article adds points to all key phrases associated to that article. Popular key phrases show us issues for topics people care most about.
```
Comment (JackyChiu) [inline README.md]: *an* article 😀

Comment (JackyChiu) [inline README.md]: Did you mean issues/topics? Instead of Issues for topics

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
-# Wall Of Memes
+CU-News
+===================
+
+CU-News is a web app that displays news articles from multiple reputable sources and finds which news articles Carleton students care about, with analysis of trends.
```
Comment (JackyChiu) [inline README.md]: Carleton students care about *through a voting system* 😀

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
-# Wall Of Memes
```
Comment (JackyChiu) [inline README.md]: *Moment of silence*

File: README.md [hunk 5]
```diff
@@ -1 +1,27 @@
...
+-------------
+
+Each article has key phrases extracted from it and upvoting a article adds points to all key phrases associated to that article. Popular key phrases show us issues for topics people care most about.
+
+----------
+
+Analytics
+-------------------
+
+The popularity of all key phrases can be graphed over time and can be compared to each other
```
Comment (kirinrastogi) [inline README.md]: Revised sentence: 

```The popularity of all key phrases can be graphed over time, and key phrases can be compared to each other.```

You might want to add 'graphically' or similar to the end of this sentence, depending on how we will compare them.

**Comment by VikramBombhi** [comment]
I no good at english

File: README.md (hunk 5 above)
Comment (VikramBombhi) [inline README.md]: I dont know English so im just going to trust u kirin

**Comment by VikramBombhi** [comment]
Ok guys I put in all the changes yall wanted let me know if its good to merge

**Comment by JackyChiu** [comment]
@VikramBombhi There's no new updates, and don't forget to squash your commits, then finally merge and squash

**Comment by VikramBombhi** [comment]
sorry push didnt go through, its up now. Also ill use that squash and merge button

**Comment by JackyChiu** [comment]
LGTM 👍

**Pull request merged**